uktrade/data-hub-frontend
# Refactor EditCompanyListForm to use TaskForm

## Description of change

Refactors the code for the EditCompanyList to make use of the new TaskForm component

## Test instructions

After creating a list and going to edit the list name, you should be able to edit the list name successfully.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
